core/scss/utilities/functions/modular-scale/_get-modular-spacing-multiplier.scss

```diff
@@ -0,0 +1,16 @@
+@charset "UTF-8";
+
+//
+// get-modular-spacing-multiplier($step)
+//
+// Get a value out of the modular-spacing-multiplier map.
+// Returns the value of the $modular-spacing-multiplier variable at a step(integer) on the scale.
+// The default for $step is 0, which returns a multiplier value of 1.
+//
+// $step - Integers from -4 to 5 including 0
+//
+// Style guide: Functions.ModularScale.get-modular-spacing-multiplier
+//
+@function get-modular-spacing-multiplier($step: 0) {
```

Comment:
I think this needs some error handling. What if I am a bad user and send in something that isn't -4 through 5?

Reply:
``` 
 // If user passes through non-valid `$step`, return `null`
    @if $step < -4 {
      @return 'null';
    }
    @if $step < 5 {
      @return 'null';
    }
```

Reply:
```
// If user passes through non-valid `$step`, return `null`
@if $step < -4 {
  @error 'Step`#{$step}` is not a valid modular value';
}
@if $step < 5 {
  @error 'Step`#{$step}`  is not a valid modular value;
}
```

Reply:
In this case, you would not want to return null as that could have possible downstream problems and actually make it more difficult to debug. Errors are useful and in this case, you want to throw an error and stop compilation if you get a bad value.

also, you can combine your if statements here with an `or` operator. 

https://stackoverflow.com/questions/17225229/using-multiple-conditions-and-in-sass-if-statement/17226136

Reply:
That makes sense. Also, thanks for the link; super useful operators there.

Would we just want to throw an error then (something closer to the second example above)?

Reply:
Thanks guys! I'll add the error handling.